libs/types/src/tokens.rs

```diff
@@ -204,6 +204,48 @@ pub struct CustomMetadata {
 
 	/// Whether an asset can be used as a currency to fund Centrifuge Pools.
 	pub pool_currency: bool,
+
+	/// The ways this token is cross-chain transferable
```

Comment:
But the comment about is referring to a `bool`, where `whether` makes sense. `transferability` is not _just_ "whether" it's cross-chain transferable but the ways that it might be so.